src/mesh.rs

```diff
@@ -114,6 +137,24 @@ impl Mesh {
         let msg = Operation::SetMaterial(material);
         let _ = self.object.tx.send((self.object.node.downgrade(), msg));
     }
+
+    /// Bind a skeleton to the mesh.
+    pub fn set_skeleton(
+        &mut self,
+        skeleton: Skeleton,
+    ) {
+        let msg = Operation::SetSkeleton(skeleton);
+        let _ = self.object.tx.send((self.object.node.downgrade(), msg));
+    }
+
+    /// Bind a set of morph targets to the mesh.
+    pub fn set_targets(
+        &mut self,
+        targets: ArrayVec<[Target; MAX_TARGETS]>,
```

Comment:
Ideally we'd take an `Iterator<Item = Target>` here but since there's a hard limit on the number of targets I wasn't so sure.

Reply:
Uhm, maybe we should not expose `ArrayVec` to user?

Reply:
Not sure; it's up for debate. An alternative is to take `IntoIterator`.